<?php

use Symfony\Component\DependencyInjection\Argument\RewindableGenerator;
use Symfony\Component\DependencyInjection\Exception\RuntimeException;

// This file has been auto-generated by the Symfony Dependency Injection Component for internal use.
// Returns the private 'security.authentication.provider.guard.secured_area' shared service.

include_once $this->targetDirs[3].'/vendor/symfony/security/Core/Authentication/Provider/AuthenticationProviderInterface.php';
include_once $this->targetDirs[3].'/vendor/symfony/security/Guard/Provider/GuardAuthenticationProvider.php';
include_once $this->targetDirs[3].'/vendor/symfony/security/Core/User/UserCheckerInterface.php';
include_once $this->targetDirs[3].'/src/Security/UserChecker.php';

return $this->privates['security.authentication.provider.guard.secured_area'] = new \Symfony\Component\Security\Guard\Provider\GuardAuthenticationProvider(new RewindableGenerator(function () {
    yield 0 => ($this->privates['App\\Security\\TokenAuthenticator'] ?? $this->load('getTokenAuthenticatorService.php'));
}, 1), ($this->privates['security.user.provider.concrete.chain_provider'] ?? $this->load('getSecurity_User_Provider_Concrete_ChainProviderService.php')), 'secured_area', ($this->privates['App\\Security\\UserChecker'] ?? ($this->privates['App\\Security\\UserChecker'] = new \App\Security\UserChecker())));
